src/libultra/io/visetspecial.c

```diff
@@ -1,38 +1,55 @@
 #include "global.h"
+#include "ultra64/viint.h"
 
+/**
+ * Configures VI "special features" to be applied on the next context swap.
+ *
+ * "Special features" refer to the mode bits in the Video Interface control register that enable effects such as gamma
+ * correction, gamma dither, dither filtering, anti-aliasing filtering and divot filtering. Configuring the same
+ * setting ON and OFF in the same call will result in OFF taking precedence.
+ *
+ * Any unrecognized bits will be ignored. Note that this is very intentional as in early revisions of retail N64
+ * hardware setting bit 5 in the `features` field of OSViContext may cause physical damage to the console once it is
+ * fed to VI_CONTROL_REG on next context swap.
```

Comment:
You might as well refer to the place in the docs that mentions this here, I think.

Reply:
There is no manual page that discusses this afaik